configurations/app_dashboard.json

```diff
@@ -0,0 +1,29 @@
+{
+  "apps": [
+    {
+      "appName": "Twake",
```

Comment:
Do we need to care about multiple languages? @hoangdat

Reply:
no, app name is unique in all languages